docs/UserGuide.md

```diff
@@ -4,7 +4,7 @@ title: User Guide
 ---
 ## Introduction
```

Comment:
![image](https://user-images.githubusercontent.com/69676081/140062239-940b1baf-b53b-41e8-8907-77b42c18c44e.png)

This shortcut does not seem to work

Reply:
Fixed!

Reply:
Fixed!